docs/getting_started.rst

```diff
@@ -6,9 +6,9 @@ Getting started
 Requirements
 ------------
 
-* Python (3.7, 3.8, 3.9, 3.10)
-* Django (2.2, 3.1, 3.2, 4.0)
-* Django REST Framework (3.10, 3.11, 3.12, 3.13)
+* Python (3.7, 3.8, 3.9, 3.10, 3.11)
+* Django (2.2, 3.1, 3.2, 4.0, 4.1)
```

Comment:
We do not test Django 2.2 and 3.1 in CI, so I would not document these as supported. I've fixed this in #711.

Reply:
I've removed 2.2 :)